plugins/inputs/dpdk/dpdk_utils.go

```diff
@@ -39,6 +39,43 @@ func getParams(command string) string {
 	return command[index+1:]
 }
 
+func (dpdk *dpdk) getDpdkInMemorySocketPaths() []string {
+	filePaths := dpdk.socketGlobPath.Match()
+
+	var results []string
+	for _, filePath := range filePaths {
+		fileInfo, err := os.Stat(filePath)
+		if err != nil || fileInfo.IsDir() || !strings.Contains(filePath, dpdkSocketTemplateName) {
+			continue
+		}
+
+		if isInMemorySocketPath(filePath, dpdk.SocketPath) {
+			results = append(results, filePath)
+		}
+	}
+
+	return results
+}
```

Comment:
Can we please move this to the `dpdk.go` file as it is a member of that class...